### PR TITLE
buku: 4.2.2 -> 4.3, introduce new dependency

### PR DESCRIPTION
**Pull request by matthiasbeyer**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [x] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by timokau** [comment]
@GrahamcOfBorg eval
https://github.com/NixOS/nixpkgs/pull/83529

**Review by Br1ght0ne — APPROVED** [review]
diff LGTM
`buku --help` works
```
[6 built, 92 copied (147.0 MiB), 26.9 MiB DL]
https://github.com/NixOS/nixpkgs/pull/84097
3 package built:
buku python37Packages.flask-reverse-proxy-fix python38Packages.flask-reverse-proxy-fix
```

File: pkgs/development/python-modules/flask-reverse-proxy-fix/default.nix [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  # master fixes flask import syntax and has no major changes
+  # new release requested: https://github.com/sublee/flask-silk/pull/6
+  src = fetchFromGitHub {
+    owner = "antarctica";
+    repo = "flask-reverse-proxy-fix";
+    rev = "v${version}";
+    sha256 = "1jbr67cmnryn0igv05qkvqjwrwj2rsajvvjnv3cdkm9bkgb4h5k5";
+  };
+
+  disabled = (!isPy3k);
```
Comment (jonringer) [inline pkgs/development/python-modules/flask-reverse-proxy-fix/default.nix:21]:
```suggestion
  disabled = !isPy3k;
```

File: pkgs/development/python-modules/flask-reverse-proxy-fix/default.nix [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+  meta = with stdenv.lib; {
+    description = "Python Flask middleware for applications running under a reverse proxy";
+    maintainers = with maintainers; [ matthiasbeyer ];
+    homepage = "https://github.com/antarctica/flask-reverse-proxy-fix";
+
+    license = {
+      fullName = "Open Government Licence";
+      url = "http://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/";
+    };
```
Comment (jonringer) [inline pkgs/development/python-modules/flask-reverse-proxy-fix/default.nix:35]: did you want to add this to lib.licenses?

Comment (matthiasbeyer) [reply pkgs/development/python-modules/flask-reverse-proxy-fix/default.nix:35]: no.

**Review by jonringer — APPROVED** [review]
diff LGTM
commits LGTM
has tests :+1: 

```
https://github.com/NixOS/nixpkgs/pull/84097
3 package built:
buku python37Packages.flask-reverse-proxy-fix python38Packages.flask-reverse-proxy-fix
```

**Pull request merged**